mlrun/config.py

```diff
@@ -465,9 +465,18 @@
     },
     "default_workflow_runner_name": "workflow-runner-{}",
     "sidecar": {
```

Comment:
No need for this hierarchy - the fact that it's a sidecar isn't really interesting or relates to the purpose of these configurations. I would keep the `logs_collector` hierarchy and promote it one level up.